atlasdb-config/src/main/java/com/palantir/atlasdb/factory/AtlasDbDialogueServiceProvider.java

```diff
@@ -123,7 +123,7 @@ public ConjureTimelockService getConjureTimelockService() {
         return new TimeoutSensitiveConjureTimelockService(shortAndLongTimeoutServices);
     }
 
-    MultiClientConjureTimelockServiceBlocking getMultiClientConjureTimelockService() {
+    public MultiClientConjureTimelockServiceBlocking getMultiClientConjureTimelockService() {
```

Comment:
There are three places where the break existed. For each one, they can either:
- Construct a conjure client using internal server lib
- Have an instance of `AtlasDbDialogueServiceProvider`

This is why I expose this here as public